plugins/network/upnpc_

```diff
@@ -37,21 +39,38 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 =cut
 
+if [ "${MUNIN_DEBUG:-0}" = 1 ]; then
+    set -x
+fi
+
+if ! command -v upnpc >/dev/null; then
```

Comment:
This test should be part of the `autoconf` handling.
At the moment a call for `autoconf` results in an invalid response string (`upnpc not found ...`) instead of `no (upnpc not found)`.

Probably you should move the data retrieval (`DATA=$(upnpc -s)`) down to the `config` and `fetch` operation in order to avoid the complexities auf combining `autoconf` and central data collection?